### PR TITLE
Hybrid: validate CVR votes aren't more than contest settings votes

### PR DESCRIPTION
**Pull request by jonahkagan**
If the audit admin enters total vote counts for a contest choice that are less than the number of votes for that choice in the CVRs, something is wrong!

### CODE REVIEW

File: server/api/sample_sizes.py [hunk 1]
```diff
@@ -93,6 +93,12 @@ def validate_hybrid_manifests_and_cvrs(contest: Contest):
             f" times the number of votes allowed ({contest.votes_allowed})"
         )
 
+    if any(count.non_cvr < 0 for count in vote_counts.values()):
```
Comment (arsalansufi) [inline server/api/sample_sizes.py:96]: How exactly does a `count.non_cvr < 0` mean that the CVRS contain more votes than were entered in the contest settings?

File: server/tests/hybrid/test_hybrid.py [hunk 2]
```diff
@@ -720,6 +720,49 @@ def test_hybrid_manifest_validation_few_non_cvr_ballots(
...
+            "name": "Contest 1",
+            "isTargeted": True,
+            "choices": [
+                {"id": str(uuid.uuid4()), "name": "Choice 1-1", "numVotes": 13},
+                {"id": str(uuid.uuid4()), "name": "Choice 1-2", "numVotes": 10},
+            ],
+            "numWinners": 1,
+            "votesAllowed": 2,
+            "jurisdictionIds": jurisdiction_ids[:2],
+        },
```
Comment (arsalansufi) [inline server/tests/hybrid/test_hybrid.py:744]: What about this setup triggers the error? For example, is it the fact that 13 is greater than 2 (votes allowed)?

**Review by arsalansufi — APPROVED** [review]
Some understanding questions but LGTM!

File: server/api/sample_sizes.py (hunk 1 above)
Comment (jonahkagan) [reply server/api/sample_sizes.py:96]: In hybrid audits, we get two pieces of data from users:
1. The CVRs
2. The total votes reported for each candidate
In order to figure out the vote counts for the non-CVR ballots, we subtract the CVR votes from the total votes. Then we have the vote counts for both the CVR ballots and the non-CVR ballots, and we can perform the hybrid audit on each segment (ballot comparison for CVR ballots, ballot polling for non-CVR ballots).

The error that happened in this case was that the non-CVR vote count was negative, which is invalid and caused an assertion failure in the audit math.

File: server/tests/hybrid/test_hybrid.py (hunk 2 above)
Comment (jonahkagan) [reply server/tests/hybrid/test_hybrid.py:744]: It's that 13 is less than the total votes for Choice 1-1 in the CVR. I can try to make that a bit more clear

File: server/api/sample_sizes.py (hunk 1 above)
Comment (arsalansufi) [reply server/api/sample_sizes.py]: Got it, thanks for explaining!

Negative non-CVR count --> more CVRs than total votes ✅

**Pull request merged**